src/lib/trips.js

```diff
@@ -131,10 +133,45 @@ export const getSectionsInfo = trip => {
   ).filter(Boolean)
 }
 
+export const getSectionsFormatedInfo = trip => {
+  const sections = getSectionsInfo(trip)
+
+  return sections.map(section => {
+    return {
+      ...section,
+      distance: `${formatDistance(section.distance)}`,
+      duration: `${humanizeDuration(section.duration * 1000, {
+        delimiter: ' ',
+        largest: 2,
+        round: true,
+        units: ['h', 'm'],
+        language: 'shortEn',
+        languages: {
+          shortEn: {
+            h: () => 'h',
+            m: () => 'min',
+            s: () => 'sec',
+            ms: () => 'ms'
+          }
+        }
+      })}`,
+      averageSpeed: `${Math.round(section.averageSpeed)} km/h`
+    }
+  })
+}
+
 export const getStartDate = trip => {
   return new Date(trip.properties.start_fmt_time)
 }
 
+export const getEndDate = trip => {
+  return new Date(trip.properties.end_fmt_time)
+}
+
+export const getTime = date => {
+  return format(date, 'HH[h]mm')
```

Comment:
You should use the format() returned by useI18n() : 

const { f } = useI18n(). 

Here I don't know how is handled the multilanguage?

Reply:
J'ai modifié l'approche (voir le dernier commit) https://github.com/cozy/coachCO2/pull/19/commits/c65000c6e43de1ceab862d8946677f93ddb83c03 pas sûr que ce soit le top... qu'en penses-tu ?